File: battleship.rb
#!/usr/bin/env ruby
# 'The Game Replay: Game Part 1' http://www.youtube.com/watch?v=1z7vgnslBkk
# resource file?
# operating system dependent style
require 'Qt4'
require 'gamewindow'
app = Qt::Application.new ARGV
window = GameWindow.new
window.show
app.exec

File: battleship.cc
#include <QtGui/QApplication>
#include "gamewindow.hh"

int main(int argc, char *argv[])
{
  QApplication app(argc, argv, QApplication::GuiClient);
  // app.setStyle("windows");
  // app.setStyle("macintosh");
  app.setStyle("plastique");
  GameWindow window;
  window.show();
  return app.exec();
};


